
---
nav_msgs/OccupancyGrid map
std_msgs/Int16 cell_occ_value
